Specs/Scene/Batched3DModel3DTileContentSpec.js

```diff
@@ -103,11 +103,20 @@ defineSuite([
     });
 
     it('logs deprecation warning for use of BATCHID without prefixed underscore', function() {
-        var deprecationWarningSpy = jasmine.createSpy(deprecationWarning);
-        return Cesium3DTilesTester.loadTileset(scene, withBatchTableUrl).then(function(tileset) {
-            expect(deprecationWarningSpy).toHaveBeenCalled();
-            Cesium3DTilesTester.expectRenderTileset(scene, tileset);
-        });
+        var called = false;
+        var originalDeprecationWarning = Batched3DModel3DTileContent._deprecationWarning;
```

Comment:
Instead of overriding the function manually, use `spyOn(Batched3DModel3DTileContent, '_deprecationWarning');` and then you can use `expect(Batched3DModel3DTileContent._deprecationWarning).toHaveBeenCalled()` in the `then` and there's no reason to have an otherwise or have to manually set it back.